postgrescontroller/cmd/postgrescontroller/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"context"
+	"runtime"
+
+	stub "github.com/demo/postgrescontroller/pkg/stub"
```

Comment:
When you included this file under labs repo, shouldn't the path refer to that?

stub "github.com/cloud-ark/labs/postgresscontroller/pkg/stub"?

Also, lets change the name of the folder to "postgres-with-operator-sdk"